checknoglobals/check_no_globals.go

```diff
@@ -96,6 +96,31 @@ func looksLikeError(i *ast.Ident) bool {
 	return strings.HasPrefix(i.Name, prefix)
 }
 
+// hasEmbedComment returns true if the AST identifier has
+// a '//go:embed ' comment, or false otherwise.
+func hasEmbedComment(i *ast.Ident) bool {
+	if i.Obj == nil {
+		return false
+	}
+
+	spec, ok := i.Obj.Decl.(*ast.ValueSpec)
+	if !ok {
+		return false
+	}
```

Comment:
❓ What does this condition guard again?

Reply:
I always make a type assertion. You never know...